plugins/woocommerce-admin/client/marketing/components/product-icon/icons/library/zapier.js

```diff
@@ -0,0 +1,8 @@
+const zapier = (
+	<img
+		src={ `${ window.wcSettings.wcAssetUrl }images/marketing/zapier.png` }
+		alt="lala"
```

Comment:
I guess instead of "lala" we can set alt as zapier?

Reply:
Same goes for the other images

Reply:
Thanks, it was a copy-and-paste mistake. Fixed in [`573a6bf` (#34415)](https://github.com/woocommerce/woocommerce/pull/34415/commits/573a6bf66ce736fbe3f40f031345c375f57b2c7d).